src/map/instance.hpp

```diff
@@ -100,7 +100,7 @@ struct s_instance_db {
 
 class InstanceDatabase : public TypesafeYamlDatabase<int32, s_instance_db> {
 public:
-	InstanceDatabase() : TypesafeYamlDatabase("INSTANCE_DB", 1) {
+	InstanceDatabase() : TypesafeYamlDatabase("INSTANCE_DB", 2) {
```

Comment:
Make it minimum 1, max 2.
This way version 1 can still be read and only get warning.